.gitignore

```
@@ -7,3 +7,20 @@ target/
 # IDE and editor configs
 /.idea/
 .vscode/
+
+
+# Keycloak chart env and package files
+infrastructure/keycloak-chart/**/*.tgz
+
+# Temporary scripts
+*.tmp.sh
+
+# Terraform state and cache
+terraform.tfstate
+terraform.tfstate.backup
+.terraform/
+**/.terraform.lock.hcl
+
+# Docker Compose environment file (generated from config.yaml)
+.env
```

```suggestion
.env*
```
This will also ignore the temporary `.env` files.
Please ensure only a single newline remains in this file.